library/build.gradle

```diff
@@ -85,6 +85,7 @@ dependencies {
   compile "com.mapzen.tangram:tangram:$tangram_version"
   compile 'com.mapzen.android:lost:1.1.1'
   compile 'com.mapzen:on-the-road:0.8.4'
+  compile('com.mapzen.android:pelias-android-sdk:0.7.2-SNAPSHOT')
```

Comment:
I'm hoping we won't have to worry about excluding the javax `annotation-api` dependency here like we currently do in EM. I believe we ran into issues a while back with two different versions included by two different libraries.

https://github.com/mapzen/eraser-map/blob/master/app/build.gradle#L133

If indeed we don't need the exclude here we can drop the `( )` around the module descriptor.